User guide updates membrane (#1933)
OpenFreeEnergy/openfe: docs/guide/setup/defining_protocols.rst

   protocol = openmm_rfe.RelativeHybridTopologyProtocol(settings)

Adaptive Settings
~~~~~~~~~~~~~~~~~

In addition to the ``.default_settings()`` method, some protocols
provide an ``_adaptive_settings`` method. This method generates recommended settings
based on properties of the input :class:`.ChemicalSystem`\s and, where required, the :class:`.AtomMapping`.

For example::

   from openfe.protocols import openmm_rfe

   settings = openmm_rfe.RelativeHybridTopologyProtocol._adaptive_settings(
       stateA=stateA,
       stateB=stateB,
       mapping=mapping,
   )

   protocol = openmm_rfe.RelativeHybridTopologyProtocol(settings)

The adaptive settings may modify parameters based on properties of the input systems.
For example (:class:`.RelativeHybridTopologyProtocol`):

* Transformations involving a change in net charge use a larger number of lambda windows and longer production simulations.
* If both states contain a :class:`.ProteinComponent`, the solvation padding is set to 1 nm.

If an ``initial_settings`` object is provided, the adaptive settings are based on a copy of the provided

> ``initial_settings`` rather than the default settings.

Reviewer: ```suggestion
Optionally, the `_adaptive_settings` method can be provided a pre-existing settings object via the ``initial_settings`` argument. If provided, an adapted copy of these settings will be returned instead
of using the default settings.
```
Maybe a bit of a nit - just thinking this rewording might be slightly more informative.

* If both states contain a :class:`.ProteinComponent`, the solvation padding is set to 1 nm.

> Optionally, the `_adaptive_settings` method can be provided a pre-existing settings object via the ``initial_settings`` argument. If provided, an adapted copy of these settings will be returned instead

Reviewer 2: ```suggestion
Optionally, you can pass a preexisting settings object to the ``_adaptive_settings`` method via the ``initial_settings`` argument. If provided, an adapted copy of these settings will be returned instead
```

edit to remove the passive voice, hopefully this is clearer.

.. warning::

>   The ``_adaptive_settings()`` method is experimental and subject to change.

Reviewer: This is great. As a small nit, it might be good to move that to the top so that it's the first thing folks read.

Author: Done!
